src/test/app.spec.js

```diff
@@ -0,0 +1,7 @@
+const findFiles = require('../app');
+
+describe('archivo md', () => {
+    it('deberia leer un archivo md', () => {
```

Comment:
Te recomiendo completar la descripción 
```suggestion
describe('file .md', () => {
    it('force read md file, () => {
```